source/guides/administration.rst

```diff
@@ -56,6 +57,7 @@ These resources will help you get started with your Mattermost self-hosted or Cl
 * **Manage team and channel members** - :doc:`Manage team and channel members </manage/team-channel-members>` synchronization, moderation, and membership settings.
 * **Customize branding** - Change Mattermost branding, site name, and description with :doc:`custom branding tools </configure/custom-branding-tools>`.
 * **Compliance tools** - Extract data from Mattermost for :doc:`eDiscovery </comply/electronic-discovery>` use in legal cases, create :doc:`Compliance monitoring </comply/compliance-monitoring>` reports for query and download actions, or :doc:`Compliance export </comply/compliance-export>` reports for channel history actions, set custom :doc:`data retention policies </comply/data-retention-policy>`, and set :doc:`custom terms of service </comply/custom-terms-of-service>` for team members.
+* **Notify Admin** - Keep track of your users' :doc:`feature requirement needs </upgrade/notify-admin>`.
```

Comment:
This is another instance of a docs organizational change in-progress that will impact this PR. Sorry! When https://github.com/mattermost/docs/pull/5885 is approved and merged, administrative content for system admins will be re-organized to be easier to discover and navigate. 

This looks like a great candidate for the new [Manage your Mattermost workspace](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/5885/guides/administration.html#manage-your-mattermost-workspace) page. If the notify admin functionality is available for both self-hosted and Cloud deployments, it'll fit well in the first section of links.

I recommend that we merge https://github.com/mattermost/docs/pull/5885 into production first, then merge latest master into this PR, then move this new feature to its new home.

Reply:
@cwarnermm, that sounds great. This is in the release branch - when do you reckon #5885 will be merged? I can pull this from the release branch if needed. I am awaiting some additional feedback so it would work fine with my timeline.

Reply:
Good question, @justinegeffen. https://github.com/mattermost/docs/pull/5885 is currently in review with the PM team. We're currently considering it as a candidate for the v7.5 release, but that decision isn't finalized yet.

We can certainly merge in this PR without https://github.com/mattermost/docs/pull/5885 if needed. I recommend creating a follow-on card on our board to update links as needed once https://github.com/mattermost/docs/pull/5885 is merged in.

Reply:
Card created. :)